# try:
	# print(x)
# except:
		# print("x is not defined")



# try:
    # print('hello World')
# except:
    # print("something is wrong")
# else:
    # print("nothing wrong")
# finally:
    # print("finally try is completed")    
    
    
#try:
 #   print(x)
#except Exception:
 #   print("wrong of x")
  #  print(Exception)
#finally:
 #   print("finally printed")    

# try:
    # s=open("G:\\Notepadd_python\\practefile.txt","a")
    # s.write("Trilochan")

    # s=open("G:\\Notepadd_python\\practefile.txt","r+")
    # print(s.readlines())
# except:
    # print("sivaram is not entered")
   
# finally:
    # s.close()
#try:
   # a = int(input("Enter a:"))    
   # b = int(input("Enter b:"))    
  #  c = a/b  
  #  print("a/b = %d"%c)    
# Using Exception with except statement. If we print(Exception) it will return exception class  
#except Exception:    
   # print("can't divide by zero")    
   # print(Exception)  
#else:    
   # print("Hi I am else block") 
    
    
try:
    a=5
    b=5
    print (a/b)

except Exception:
    print("not printe")
    print(Exception)

else:
    print("else part")

finally:
    print("final one")


        